test/api/v3/unit/libs/collectionManipulators.test.js

```diff
@@ -0,0 +1,71 @@
+import {
+ removeElementFromArray,
+} from '../../../../../website/src/libs/api-v3/collectionManipulators';
+
+describe('Collection Manipulators', () => {
+  describe('removeElementFromArray', () => {
+    it('removes item from specified array on document', () => {
```

Comment:
Is this supposed to test mongo documents? If not, don't we need such a test, because we expect this function to mark the array as modified?